apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -544,7 +558,7 @@ class CensusForm extends Component {
               </div>
               <input
                 type="text"
-                name="name_s"
+                name="submitter_name"
```

Comment:
`submitter_name` and `pledged` are much clearer than the previous names - thanks for updating 🙂